locales/ar-arb/messages.po

```diff
@@ -464,13 +468,13 @@ msgid "Enter tooltip target"
 msgstr "أدخل هدف تلميح الأداة"
 
 msgid "Error on item %1$s : %2$s"
-msgstr "خطأ في البند ٪ 1 $ s:٪ 2 $ s"
+msgstr "خطأ في البند %1$s : %2$s"
```

Comment:
I'm no expert at all, but just raising your attention here to the fact the first `%1$s` placeholder has been reduced to `1%`, while the second one was left as is. Was that really expected?

Reply:
It seems to be a glitch on the GitHub diff feature, the translation looks fine:

![image](https://user-images.githubusercontent.com/1084316/152958974-876123bc-eb59-4b0e-9a89-3b81f247902e.png)